packages/python-sdk/e2b/sandbox_sync/process/main.py

```diff
@@ -138,6 +137,7 @@ def run(
         timeout: Optional[float] = 60,
         request_timeout: Optional[float] = None,
     ):
+        """Run command"""
```

Comment:
Will the API generation work with the overloads? Maybe just having the base method with all the formats enumerated is enough — in some cases like `commands.run` where the overload changes the behavior it might be tricky because the docstring for each overload explains the behavior.

Reply:
The way it generates the docs is on a per package bases, so part of the targets are `packages/python-sdk/e2b/sandbox_{async, sync}` , as long as it is not overloaded inside those directories there should be no conflict.